hw/src/pages/ReposSearchPage/ReposSearchPage.tsx

```diff
@@ -0,0 +1,69 @@
+import SearchIcon from '@components/SearchIcon'
+import Button from '@components/Button'
+import RepoTile from '@components/RepoTile'
+import Input from '@components/Input'
+import GitHubStore from '../../store/GitHubStore'
+import { ApiResponse } from 'src/shared/store/ApiStore/types'
+import { RepoItem } from 'src/store/GitHubStore/types'
+
+import './ReposSearchPage.css'
+import { useCallback, useState } from 'react'
+
+const ReposSearchPage = () => {
+    const [value, setValue] = useState('');
+    const [isLoading, setIsLoading] = useState(false);
+    const [data, setData] = useState([{
+        id: 123,    
+        name: 'kts-frontend-winter-school',
+        owner: {
+            login: 'ktsstudio',
+            avatar_url: ''
+        },
+        created_at: '21 Jul',
+        stargazers_count: 123,
+    }]); 
```

Comment:
лучше загрузить при открытии страницы репозитории с аккаунта `ktsstudio`
```suggestion
    const [data, setData] = useState([]); 
```